docs/guides/tools/metamask.md

```diff
@@ -0,0 +1,61 @@
+<!--
+order: 1
+-->
+
+# MetaMask: Connecting to Ethermint
```

Comment:
how does this guide relate to the one on `guides/keys-wallets/`?

Reply:
wondering if we should merge them

Reply:
Merged